hs/src/LedgerState.hs

```diff
@@ -155,13 +161,29 @@ validInputs (TxWits tx _) l =
     else Invalid [BadInputs]
   where unspentInputs (UTxO utxo) = Map.keysSet utxo
 
+-- |Minimum fee calculation
+minfee :: PtrlConsts -> TxWits -> Coin
+minfee pc txwit = minfeeA pc * x + minfeeB pc
+  where x = Coin $ toEnum $ length (show txwit)
```

Comment:
I think we should add this reasoning and actual formula for this in the documentation of the function. This will definitely influence test cases, generators etc.

Reply:
in the spec I left it abstract, since I didn't think it mattered to much what the calculation was.  here I based it loosely on how it is done today (where `length (show txwit)` is just a simple bytes-like calculation).  But I'm happy making it explicit in the spec, or at least saying it's this ax+b calculation (and not worrying about an actual encoding here).

Reply:
I made an issue: #102